src/components/dictionaryConcepts/containers/DictionaryConcepts.jsx

```diff
@@ -109,6 +110,7 @@ export class DictionaryConcepts extends Component {
         this.state.type,
         this.state.typeName,
         this.state.collectionName,
+        this.state.searchInput,
```

Comment:
Same to this. Try to use destructuring.

Reply:
Thanks for the feedback, @kmunene . I have made the update.